### DISCUSSION TITLE
How to output a spyrmsd.molecule.Molecule object

### DISCUSSION THREAD

**Discussion by Feya** [RMeli/spyrmsd#93] [Q&A] [answered]
If we want to load a Molecule, we may use the Molecule.from_rdkit(mol). 
But  If we want to output a spyrmsd.molecule.Molecule object as a .sdf for example, how shoule I do this?
I've tried to use RDkit SDWriter
```
writer = SDWriter("outputpath.sdf")
writer.write(mol)
``` 
it doesn't work becuase the SDWriter is supposed to apply to rdkit.Chem.rdchem.Mol object, not spyrmsd.molecule.Molecule object.

So my question is how could I output a spyrmsd.molecule.Molecule object to sdf file?  
If there is no direct method, maybe we could transfer the spyrmsd.molecule.Molecule object to a rdkit.Chem.rdchem.Mol object first (without changing any of the conformer's coordinates) and then output using rdkit SDWriter. How shuold I do that?

Thank you very much!

**Comment by Feya** [comment]
Oh I realized that the initial rdkit mol is good to use. I don't have to find a way to convert the spyrmsd.molecule.Molecule object to rdkit mol.

**Reply by RMeli** [reply, marked as answer]
Hello. As you already realized, the initial RDKit molecule is good to use. `spyrmsd` does not change the molecules (the only exception being the `Molecule.strip()` function, which actually removes the hydrogen atoms from the molecular graph). This is because, for minimized RMSD, we use the QCP method. This method explicitly avoids the computation of the rigid body rotations required to superimpose two molecules.

Having functionality in `spyrmsd` to actually superimpose the molecules has been in the back of my mind for a while, but I never had the time for implementing it unfortunately.

**Comment by davide-grheco** [comment]
Having the functionality to produce the superimposed structure would be very useful.

**Reply by RMeli** [reply]
@davide-grheco, I totally agree. Unfortunately, as mentioned above, I don't have the bandwidth to work on this additional feature for the foreseeable future.
